README.md

```diff
@@ -72,6 +72,10 @@ Get in touch with the team and the community:
 
 For users that can't access our CDN there is a [script](https://github.com/scania-digital-design-system/tegel/blob/develop/packages/core/static-assets-copying.md) for copying over static assets(fonts, logos).
 
+## Testing Guide for Tegel
+
+[This](./docs/TESTING_GUIDE.md) guide provides developers with best practices and guidelines on how and when to write unit tests for components within our design system. Our aim is to ensure that each component is reliable, maintainable, and functions as intended.
```

Comment:
Just a thought, but maybe we should not have the same text in the link here pointing to the document?
So maybe just something like this suffice:

```suggestion
Refer to our [Testing Guide](./docs/TESTING_GUIDE.md) for best practices on writing unit tests for components in our design system.

[This](./docs/TESTING_GUIDE.md) guide provides developers with best practices and guidelines on how and when to write unit tests for components within our design system. Our aim is to ensure that each component is reliable, maintainable, and functions as intended.
```
?

Reply:
I think this makes a lot of since 👍